chsdi/templates/htmlpopup/oerebkataster.mako

```diff
@@ -52,7 +54,28 @@ if not 'oereb_webservice' in c['attributes'].keys():
     % if c['attributes']['oereb_webservice'] != None and c['attributes']['bgdi_status'] == 0:
         <tr>
             <td class="cell-left">${_('ch.swisstopo-vd.stand-oerebkataster.oereb_webservice')}</td>
-            <td><a target="_blank" href="${c['attributes']['oereb_webservice']}${path}${c['attributes']['egris_egrid']}">PDF</a></td>
+            <td id="${c['featureId']}">Loading link to PDF ...</td>
         </tr>
+       <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+       <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js"></script>
```

Comment:
This is a use case for an iframe tooltip since you load javascript :see_no_evil:

Reply:
I will! thanks Loic